preload/preload-invalid-resources.html

```diff
@@ -0,0 +1,34 @@
+<script src="/resources/testharness.js"></script>
```

Comment:
Add a doctype, as we're not relying on quirksmode for this test

Reply:
Done

Reply:
I think you added one in the other html, but not this one